node/_process/streams.mjs

```diff
@@ -101,9 +101,27 @@ export const stdout = stdio.stdout = createWritableStdioStream(
   "stdout",
 );
 
+function _adjustHighWaterMark() {
+  if (Deno.isatty?.(Deno.stdin?.rid)) return 0;
+
+  try {
+    // TODO(PolarETech): On Windows, isFile() returns true even for a stream,
+    // so it cannot distinguish a stream from a file.
+    // When stdin is a stream, it should return `undefined` (= 16 * 1024), not 64 * 1024.
+    if (Deno.fstatSync(Deno.stdin?.rid).isFile) return 64 * 1024;
+  } catch (_) {
+    // Avoid error that occurs when stdin is null on Windows.
+    return 64 * 1024;
+  }
+
+  // TODO(PolarETech): On Linux and Mac, 64 * 1024 should be returned when stdin is null.
```

Comment:
This seems weird to me, is there any more info you can add to this comment for the following:

A) why only the two?
B) why this number?

Reply:
A) The behavior is different on Linux/Mac and Windows. On Windows, it is able to return 64 * 1024 when stdin is null. (line 114)
B) 64 * 1024 is based on actual value in Node.js.

Reply:
Sure but is the value in nodejs meaningful or just some sentinal value they happened to pick? I would find a note here useful, to be honest. Probably anyone who needs to touch this area again will need to build up a lot of context, any hints will probably be useful.

Reply:
The value in Node.js is probably determined by performance and other considerations.
Regardless of the validity of the value, Deno members have indicated their policy that the implementation of the Node compat layer should be aligned with Node.js as much as possible, and I'm making an effort to follow the policy.

Reply:
I have added information to the Description that was found during the work on this PR. I will continue to update accordingly.

Reply:
No, problem the code is basically fine I'm just probing for some hints about the magic numbers which I realize are coming from an attempt to be compatible with node. Lookin good.